Content.IntegrationTests/Tests/Interaction/InteractionTest.cs

```diff
@@ -135,10 +135,16 @@ public abstract partial class InteractionTest
 - type: entity
   id: InteractionTestMob
   components:
-  - type: Body
-    prototype: Aghost
   - type: DoAfter
   - type: Hands
+    hands:
+      hand_right:
+        location: Right
+      hand_left:
+        location: Left
+    sortedHands:
+    - hand_right
+    - hand_left
```

Comment:
```suggestion
    hands:
      hand_right:
        location: Right
    sortedHands:
    - hand_right
```
The interaction test mob needs to have exactly one hand, for reasons ("so that they do not accidentally pick up deconstruction products").

You can also delete the horror that lurks on lines 236-248 now.